services/web-app/utils/use-query-state.js

```diff
@@ -57,15 +28,35 @@ export const useQueryState = (
       return null
```

Comment:
This comment on line 53 (old) 24 (new) probably needs an update now

Reply:
Removed, unnecessary